src/ngx_http_set_misc_module.c

```diff
@@ -79,6 +79,13 @@ static ndk_set_var_t  ngx_http_set_misc_set_hmac_sha1_filter = {
     2,
     NULL
 };
+
+static ndk_set_var_t  ngx_http_set_misc_set_hmac_sha256_filter = {
```

Comment:
forgot this one?

Reply:
It's not one of the two newlines previously requested... anyway, added it now...